File: SampleApp/java/SDK/app/src/main/java/jp/fluct/sample/samplefluctsdkapp/nativead/NativeAdSampleActivity.java
package jp.fluct.sample.samplefluctsdkapp.nativead;

import android.content.Intent;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

import jp.fluct.sample.samplefluctsdkapp.R;

public class NativeAdSampleActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_native_ad_sample);

        findViewById(R.id.btn_simple).setOnClickListener(v ->
                startActivity(new Intent(this, NativeAdSimpleActivity.class)));
        findViewById(R.id.btn_recycler).setOnClickListener(v ->
                startActivity(new Intent(this, NativeAdRecyclerActivity.class)));
    }
}

File: SampleApp/kotlin/app/src/main/java/jp/fluct/sample/kotlinapp/nativead/NativeAdSampleActivity.kt
package jp.fluct.sample.kotlinapp.nativead

import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.widget.Button
import jp.fluct.sample.kotlinapp.R

class NativeAdSampleActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_native_ad_sample)

        findViewById<Button>(R.id.btn_simple).setOnClickListener {
            startActivity(Intent(this, NativeAdSimpleActivity::class.java))
        }
        findViewById<Button>(R.id.btn_recycler).setOnClickListener {
            startActivity(Intent(this, NativeAdRecyclerActivity::class.java))
        }
    }
}
